config/locales/en.yml

```diff
@@ -800,6 +821,7 @@ en:
   label_information_plural: "Information"
   label_integer: "Integer"
   label_internal: "Internal"
+  label_invite_user: "Invite user"
```

Comment:
Why is that called "Invite user"?

It links to the create user form and that would be an apropriate label: "Create user".

I see that in the SaaS context there will be an invite feature, but that is nothing for on-premise OpenProject, is it?